server/services/BarChartTableService/index.js

```diff
@@ -32,37 +29,41 @@ class BarChartTableService {
   }
 
   csvTableData = () => {
+    const tableColumns = [SITE, ...this.websiteColumns.map(({ name }) => name)]
+    const tableRows = this.websiteRows.map((siteData) =>
+      tableColumns.map(
+        (columnHeader) =>
+          siteData[
+            columnHeader === SITE ? columnHeader.toLowerCase() : columnHeader
+          ]
```

Comment:
do we need to do this logic just to access the `siteData` property? Since `SITE` is a constant can we change it to match our headers and make this consistently `siteData[columnHeader]`?

Reply:
It current looks like this, a lowercase site feels and looks weird as a header, as a row of data not so much, and now that we're talking about this... I prefer the name Network for this column... 

<img width="638" alt="Screenshot 2024-03-19 at 1 24 44 PM" src="https://github.com/AMP-SCZ/dpdash/assets/19805355/3dce8505-18db-4205-a52f-bf34884d25a1">

Reply:
With the column title network it matches the graph table on the chart page significantly 
<img width="1121" alt="Screenshot 2024-03-19 at 1 28 02 PM" src="https://github.com/AMP-SCZ/dpdash/assets/19805355/6bb211ad-4db2-4820-adee-bc8dfa69fb1e">

Reply:
check out this commit, the adjustments all work 
[e61d2c6](https://github.com/AMP-SCZ/dpdash/pull/695/commits/e61d2c603020119c39c8112e6167441b83a0a61a)

Reply:
<img width="627" alt="Screenshot 2024-03-19 at 1 53 16 PM" src="https://github.com/AMP-SCZ/dpdash/assets/19805355/f794ff02-5406-49c0-89d3-b7c640d19a94">